mon-pix/app/pods/components/module/details/component.js

```diff
@@ -0,0 +1,8 @@
+import Component from '@glimmer/component';
+
+export default class ModuleDetails extends Component {
+  continueToNextGrain() {
```

Comment:
Il manque le décorateur : `@action` au dessus de la méthode

Reply:
@VincentHardouin on va le rajouter, mais pourquoi est-ce requis ? (vu que ça marche sans ^^)

Reply:
c'est donc surtout une sombre histoire de binding ? (dans cet article ça parle aussi de marquer l'intention du code ce genre de truc mais c'est pas hyper convainquant)

Reply:
Et de pouvoir une fonction et une action qui ont le même nom. [La doc](https://guides.emberjs.com/release/components/component-state-and-actions/) fait avec le décorateur, c'est plutôt pas mal de la suivre ;)